lib/vmpooler/views/dashboard.erb

```diff
@@ -1,21 +1,63 @@
-<!-- stats-vmpooler-running -->
+<!-- header -->
 
-<div class='label'>VMs running tests</div>
+<div id='header'>
+  <div class='row'>
+    <div id='logo' class='col-md-2'><embed src='/img/logo.gif' style='max-width: 100%; max-height: 100%' /></div>
+    <div id='site_name' class='col-md-10'><%= site_name %></div>
 
-<div id='stats-vmpooler-running' class='module'>
-  <div class='spinner'></div> Loading data...
+    <!-- dashboard-numbers -->
+
+    <div id='dashboard-numbers'></div>
```

Comment:
Above is `site_name` and this is `dashboard-numbers`. Can the use of `_` or `-` be picked and standardized?